pom.xml

```diff
@@ -106,8 +106,9 @@
             <plugin>
                 <groupId>org.glassfish.build</groupId>
                 <artifactId>spec-version-maven-plugin</artifactId>
-                <version>1.2</version>
+                <version>1.5</version>
                 <configuration>
+                    <specMode>jakarta</specMode>
                     <spec>
```

Comment:
What does this setting do for <specMode>?  I don't remember you setting this in the other change sets you are attempting to merge...  Maybe it was already there?  I won't mark this as holding up the approval, but I would like to understand this setting.  Thanks.

Reply:
@kwsutter.  I added this in response to https://github.com/javaee/spec-version-maven-plugin/pull/5.
See   https://github.com/eclipse-ee4j/management-api/issues/2 where I reported an error and Dmitry pointed me to the PR I listed.